<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>401</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icons/res/icons/prefs-ico.png</normaloff>:/icons/res/icons/prefs-ico.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="commonPage">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Common preferences</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QCheckBox" name="commonAutoStart">
               <property name="text">
                <string>Run at system startup</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="commonShowBalanceNotification">
               <property name="text">
                <string>Show account balance notifications</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>210</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="accountPage">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QGroupBox" name="loginGroupBox">
            <property name="title">
             <string>Login</string>
            </property>
            <layout class="QFormLayout" name="formLayout_2">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Email address:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="accountEmailAddress"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Password:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="accountPassword">
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="emailPage">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Email checking</string>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Automatic mail checking:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QCheckBox" name="mailCheckEnable">
               <property name="text">
                <string>Enabled</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Interval:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QSpinBox" name="mailCheckInterval">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>120</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Minute(s)</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Notification</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QCheckBox" name="mailNotifyShowNotification">
               <property name="text">
                <string>Show notification</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="mailNotfiyPlaySound">
               <property name="text">
                <string>Play sound</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="mailNotifyShowInbox">
               <property name="text">
                <string>Show inbox</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="webdiskPage">
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QGroupBox" name="wdSyncGroup">
            <property name="title">
             <string>%1 synchronization</string>
            </property>
            <layout class="QFormLayout" name="formLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Synchronization:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QCheckBox" name="webdiskEnableSync">
               <property name="text">
                <string>Enabled</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Interval:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QSpinBox" name="webdiskSyncInterval">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>120</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Minute(s)</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Settings:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QCheckBox" name="webdiskSyncHiddenItems">
               <property name="text">
                <string>Synchronize hidden items</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QCheckBox" name="webdiskShowNotifications">
               <property name="text">
                <string>Show notifications</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Target</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap resource="resource.qrc">:/icons/res/icons/folder-ico.png</pixmap>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="webdiskTargetFolder">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton">
                 <property name="text">
                  <string>Change...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="faxPage">
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QGroupBox" name="groupBox_6">
            <property name="title">
             <string>Fax printer</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="driverLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="driverButton">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>208</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="aboutButton">
          <property name="text">
           <string>About</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="okButton">
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelButton">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCommon"/>
   <addaction name="actionAccount"/>
   <addaction name="actionEmail"/>
   <addaction name="actionWebdisk"/>
   <addaction name="actionFax"/>
  </widget>
  <action name="actionCommon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/res/icons/prefs-ico.png</normaloff>:/icons/res/icons/prefs-ico.png</iconset>
   </property>
   <property name="text">
    <string>Common</string>
   </property>
  </action>
  <action name="actionAccount">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/res/icons/account-ico.png</normaloff>:/icons/res/icons/account-ico.png</iconset>
   </property>
   <property name="text">
    <string>Account</string>
   </property>
  </action>
  <action name="actionEmail">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/res/icons/email-ico.png</normaloff>:/icons/res/icons/email-ico.png</iconset>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </action>
  <action name="actionWebdisk">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/res/icons/webdisk-ico.png</normaloff>:/icons/res/icons/webdisk-ico.png</iconset>
   </property>
   <property name="text">
    <string>Webdisk</string>
   </property>
  </action>
  <action name="actionFax">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/res/icons/fax-ico.png</normaloff>:/icons/res/icons/fax-ico.png</iconset>
   </property>
   <property name="text">
    <string>Fax</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>commonAutoStart</tabstop>
  <tabstop>commonShowBalanceNotification</tabstop>
  <tabstop>accountEmailAddress</tabstop>
  <tabstop>accountPassword</tabstop>
  <tabstop>mailCheckEnable</tabstop>
  <tabstop>mailCheckInterval</tabstop>
  <tabstop>mailNotifyShowNotification</tabstop>
  <tabstop>mailNotfiyPlaySound</tabstop>
  <tabstop>mailNotifyShowInbox</tabstop>
  <tabstop>webdiskEnableSync</tabstop>
  <tabstop>webdiskSyncInterval</tabstop>
  <tabstop>webdiskSyncHiddenItems</tabstop>
  <tabstop>webdiskShowNotifications</tabstop>
  <tabstop>webdiskTargetFolder</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>driverButton</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>aboutButton</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionCommon</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showCommonPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAccount</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showAccountPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEmail</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showEmailPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionWebdisk</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showWebdiskPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aboutButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showAboutBox()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>savePrefs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>changeWebdiskTarget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>575</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFax</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showFaxPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>driverButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setupPrinter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>575</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showCommonPage()</slot>
  <slot>showAccountPage()</slot>
  <slot>showEmailPage()</slot>
  <slot>showWebdiskPage()</slot>
  <slot>showAboutBox()</slot>
  <slot>savePrefs()</slot>
  <slot>changeWebdiskTarget()</slot>
  <slot>showFaxPage()</slot>
  <slot>setupPrinter()</slot>
 </slots>
</ui>
